src/fabric-1.19.2/templates/rangeditem/rangeditem.java.ftl

```diff
@@ -89,9 +89,10 @@ public class ${name}Item extends Item {
 	</#if>
 
 	<#if data.shootConstantly>
-		@Override public void onUsingTick(ItemStack itemstack, LivingEntity entityLiving, int count) {
-			Level world = entityLiving.level;
-			if (!world.isClientSide() && entityLiving instanceof ServerPlayer entity) {
+		@Override
+		public void onUseTick(Level level, LivingEntity livingEntity, ItemStack itemstack, int i) {
```

Comment:
It can cause issues in some cases.
```suggestion
		public void onUseTick(Level level, LivingEntity livingEntity, ItemStack itemstack, int count) {
```